lib/v2/screens/app/interactor/viewmodels/app_bloc.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:async';
+import 'package:bloc/bloc.dart';
+import 'package:seeds/v2/screens/app/interactor/viewmodels/bloc.dart';
+import 'package:seeds/v2/screens/app/interactor/usecases/guardians_notification_use_case.dart';
+
+/// --- BLOC
+class AppBloc extends Bloc<AppEvent, AppState> {
+  StreamSubscription<bool?>? _hasGuardianNotificationPending;
```

Comment:
this should not be StreamSubscription<bool?>?
this can be StreamSubscription<bool> 

Once we change the `hasGuardianNotificationPending` to be none nullable 

I think this `StreamSubscription<bool?>? _hasGuardianNotificationPending;` can be annotated with `late` in order to about the `?`

Reply:
done 👍